lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -226,36 +227,37 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                     gpg45ProfileEvaluator.parseCredentials(
                             userIdentityService.getIdentityCredentials(vcStoreItems));
 
-            // Credential correlation failure
-            if (!userIdentityService.areVCsCorrelated(vcStoreItems)) {
-                return isF2FComplete
-                        ? buildF2FNotCorrelatedResponse(auditEventUser)
-                        : buildNotCorrelatedResponse(auditEventUser);
-            }
-
-            // Force reset
-            if (configService.enabled(RESET_IDENTITY.getName())) {
-                return buildForceResetResponse();
-            }
-
             // Check for attained vot from vtr
             var strongestAttainedVotFromVtr =
-                    getStrongestAttainedVotFromVtr(
+                    getStrongestAttainedVotForVtr(
                             clientOAuthSessionItem.getVtr(),
                             credentials,
                             vcStoreItems,
                             auditEventUser);
 
-            // vot achieved
+            // vot achieved for vtr
             if (strongestAttainedVotFromVtr.isPresent()) {
                 Vot attainedVot = strongestAttainedVotFromVtr.get();
                 ipvSessionItem.setVot(attainedVot.name());
                 ipvSessionService.updateIpvSession(ipvSessionItem);
-
+                if (attainedVot.getProfileType().equals(ProfileType.GPG45)) {
+                    if (!userIdentityService.areVCsCorrelated(vcStoreItems)) {
+                        return isF2FComplete
+                                ? buildF2FNotCorrelatedResponse(auditEventUser)
+                                : buildNotCorrelatedResponse(auditEventUser);
+                    }
+                }
```

Comment:
I think we actually need to move the correlation check within `achievedWithGpg45Profile`. Logically, correlation is actually part of forming a valid profile, and if a user has an uncorrelated GPG45 profile, they should still be able to achieve an operational profile.

It shouldn't be too hard to return the right journey event in this case, but you might have to do some refactoring to get the right audit events.